components/compliance-service/ingest/ingestic/ingestic.go

```diff
@@ -20,6 +20,18 @@ type ESClient struct {
 	initialized bool
 }
 
+var (
```

Comment:
@lancewf do you know if there's any elasticsearch related code shared btwn ingest-service and compliance? this script seems like it could be a shared constant.

though will that be true if we add a fetch to only get resources with the project to delete 🤔

Reply:
I don't know anywhere that elasticsearch code is shared between the two services. The code should be the same even with filtering for the deleted project. 

My preference would be to have the duplicate code in both services because the only reason the code is the same now is because the project field is named the same and is in the same location on the ES document.